car_company.py

```diff
@@ -23,10 +23,17 @@ def update_car(self, make, model, price):
         car = self.find_car(make, model)
         if car:
             car.price = price
+def list_cars(self):
```

Comment:
Seeems to be wrong indented, since the method should belong to the Inventory class